.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "nanshe/_vendor/_metawrap"]
```

Comment:
Somehow this file was not deleted, but only emptied. Actually deleting it as well in PR ( https://github.com/nanshe-org/nanshe/pull/448 ).